packages/mdc-auto-init/index.js

```diff
@@ -41,14 +42,18 @@ export default function mdcAutoInit(root = document, warn = CONSOLE_WARN) {
     }
 
     // TODO: Should we make an eslint rule for an attachTo() static method?
-    const component = Ctor.attachTo(node);
+    component = Ctor.attachTo(node);
     Object.defineProperty(node, ctorName, {
       value: component,
       writable: false,
       enumerable: false,
       configurable: true,
     });
   }
+
+  if (component !== null) {
```

Comment:
@yshterev I looked at the comments on the issue and discussion here. I think it was slightly different though when we discuss it internally as we thought it would be done at component level. 
So as now we understand the landscape of the question better, I think it should be proper to dispatch a custom event directly instead of using `emit` method from a single component. Could you update the PR accordingly? Thanks!